src/main/java/seedu/address/model/group/GroupList.java

```diff
@@ -4,54 +4,80 @@
 
 public class GroupList {
 
-    private static ArrayList<Group> groupList = new ArrayList<>();
+    private static ArrayList<Group> listGroup = new ArrayList<>();
```

Comment:
listGroup sounds like an action. I think we should keep to groupList as it is closer to a noun.